easyverein/models/member_member_group.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import Any
 
+from member_group import MemberGroup
```

Comment:
The import must be relative (`from .member_group import MemberGroup`) and must be at the end of the file.

Please move the import to line 73 (after the existing `member` import) like this:

```python
from .member_group import MemberGroup  # noqa: E402
```